src/components/PlaystationPage/PlaystationPage.jsx

```diff
@@ -0,0 +1,185 @@
+import { AiOutlineHeart, AiFillHeart } from "react-icons/ai";
+import { IoBagCheckOutline } from "react-icons/io5";
+import { LoadingState } from "../LoadingState/LoadingState";
+import { ErrorState } from "../ErrorState/ErrorState";
+import "./playstationPage.css";
+
+export const PlaystationPage = () => {
+  const yellowColor = "rgb(255, 149, 41)";
+  const blueColor = "rgb(0, 111, 205)";
+
+  const {
+    products,
+    ratingFilter,
+    priceFilter,
+    appState,
+    dispatch,
+  } = usePlaystation();
+  const { cart, cartDispatch } = useCart();
+  const { wishlist, wishlistDispatch } = useWishlist();
+
+  const screenWidth = useWindowSize().width;
+  const { navPhoneVisible, setNavPhoneVisibility } = useNavPhone();
+
+  const { setSnackbar } = useAlert();
+
+  useEffect(() => setNavPhoneVisibility(false), [setNavPhoneVisibility]);
+
+  const filterData = (products, price, rating) => {
+    let data = [...products];
+    if (price)
+      data = data.filter(
+        (item) => item.price >= price.low && item.price < price.high
+      );
+    if (rating) data = data.filter((item) => item.rating >= rating);
+    return data;
+  };
+  const filteredData = filterData(products, priceFilter, ratingFilter);
+
+  return (
+    <div className="container-app">
+      {screenWidth < 768 && navPhoneVisible && (
+        <NavPhone active="playstation" />
+      )}
+      <Header active="playstation" />
+      <main className="main-container-products">
+        {screenWidth < 768 ? (
+          <FilterPhone
+            dispatch={(args) => dispatch(args)}
+            ratingFilter={ratingFilter}
+            priceFilter={priceFilter}
+          />
+        ) : (
+          <FilterMenu
+            dispatch={(args) => dispatch(args)}
+            ratingFilter={ratingFilter}
+            priceFilter={priceFilter}
+          />
+        )}
+        {appState === "loading" && <LoadingState />}
+        {appState === "error" && <ErrorState />}
+        {appState === "success" && (
+          <div className="games-list">
+            <div className="product-grid">
+              {filteredData.length !== 0 &&
+                filteredData.map((item) => {
+                  return (
+                    <CardCustom key={item.id}>
+                      <CardImage image={item.coverImage} title={item.name} />
+                      <CardContent>
+                        <div>
+                          <div
+                            style={{
+                              display: "flex",
+                              justifyContent: "space-between",
+                            }}
+                          >
+                            <span className="custom-container-rating">
+                              <StarRating
+                                rating={item.rating}
+                                color={yellowColor}
+                              />
+                            </span>
+
+                            {wishlist.find(

```

Comment:
such conditions can be extracted to a function in the component called `checkWishlist` or some other name.